src/components/MAPI/workernodes/WorkerNodesCreateNodePool.tsx

```diff
@@ -114,6 +116,7 @@ function makeInitialState(
       [NodePoolPropertyField.Name]: true,
       [NodePoolPropertyField.Description]: true,
       [NodePoolPropertyField.MachineType]: true,
+      [NodePoolPropertyField.AvailabilityZones]: false,
```

Comment:
AZs are invalid by default, since the control plane nodes have to be loaded first. This will be set to `true` after the control plane nodes have been loaded successfully.